extensions/sql-migration/src/constants/helper.ts

```diff
@@ -0,0 +1,22 @@
+
+import * as azdata from 'azdata';
+import { getSqlServerName } from '../api/utils';
+import { MigrationContext } from '../models/migrationLocalStorage';
+import * as loc from './strings';
+
+export function getMigrationTargetType(migration: MigrationContext): string {
+	return migration.targetManagedInstance.type === 'microsoft.sql/managedinstances'
+		? loc.SQL_MANAGED_INSTANCE
```

Comment:
can you create, export and reference a constant like  SqlManagedInstanceAssetType constant for 'microsoft.sql/managedinstances'

Reply:
I added an enum for the asset types and updated the logic